<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>311</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>TEST LIB</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="camarasTabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="IODevice">
       <attribute name="title">
        <string>I/O</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="2" rowspan="3">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="ioDeviceSTATUSLabel">
            <property name="baseSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::Panel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="lineWidth">
             <number>7</number>
            </property>
            <property name="midLineWidth">
             <number>4</number>
            </property>
            <property name="text">
             <string>ESTATUS</string>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="ioDeviceREDLabel">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="lineWidth">
               <number>7</number>
              </property>
              <property name="midLineWidth">
               <number>4</number>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ioDeviceGREENLabel">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>80</width>
                <height>80</height>
               </size>
              </property>
              <property name="autoFillBackground">
               <bool>true</bool>
              </property>
              <property name="frameShape">
               <enum>QFrame::Panel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="lineWidth">
               <number>7</number>
              </property>
              <property name="midLineWidth">
               <number>4</number>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="ioDeviceONpushButton">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>25</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>40</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>ON</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ioDeviceOFFpushButton">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>25</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>40</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>OFF</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>248</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="guardarIODevicesPushButton1">
          <property name="text">
           <string>Guardar</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="2">
           <widget class="QToolButton" name="actualizarIODEvicestoolButton">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="ioDevicesComboBox1">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Dispositivo</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>171</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Camara1">
       <attribute name="title">
        <string>Camara1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <layout class="QFormLayout" name="formLayout">
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>14</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Tipo de Camara</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="camaraComboBox1"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Host</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="hostLineEdit1"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Puerto</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="puertoLineEdit1">
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Usuario</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="usuarioLineEdit1"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Contraseña</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="passwdLineEdit1">
              <property name="inputMethodHints">
               <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="guardarPushButton1">
              <property name="text">
               <string>Guardar</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QPushButton" name="webPushButton1">
              <property name="text">
               <string>WebAdmin</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QLabel" name="camaraLabel1">
           <property name="minimumSize">
            <size>
             <width>320</width>
             <height>240</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>320</width>
             <height>240</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>2</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Camara2">
       <attribute name="title">
        <string>Camara2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QWidget" name="layoutWidget_2">
           <layout class="QFormLayout" name="formLayout_3">
            <property name="horizontalSpacing">
             <number>6</number>
            </property>
            <property name="verticalSpacing">
             <number>14</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_16">
              <property name="text">
               <string>Tipo de Camara</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="camaraComboBox2"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Host</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="hostLineEdit2"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_18">
              <property name="text">
               <string>Puerto</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="puertoLineEdit2">
              <property name="inputMethodHints">
               <set>Qt::ImhDigitsOnly</set>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Usuario</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="usuarioLineEdit2"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_20">
              <property name="text">
               <string>Contraseña</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="passwdLineEdit2">
              <property name="inputMethodHints">
               <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="guardarPushButton2">
              <property name="text">
               <string>Guardar</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QPushButton" name="webPushButton2">
              <property name="text">
               <string>WebAdmin</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QLabel" name="camaraLabel2">
           <property name="minimumSize">
            <size>
             <width>320</width>
             <height>240</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>320</width>
             <height>240</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>2</number>
           </property>
           <property name="midLineWidth">
            <number>2</number>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
